scripts/us_census/pep/pep_by_srh/process.py

```diff
@@ -837,16 +1200,18 @@ def _consolidate_state_files():
 
 def _consolidate_county_files():
```

Comment:
Implement try/catch block in this method.

Reply:
Implemented try/catch block